src/printer.js

```diff
@@ -4359,11 +4359,17 @@ function printBinaryishExpressions(
       parts.push(path.call(print, "left"));
     }
 
-    const right = concat([
-      node.operator,
-      shouldInlineLogicalExpression(node) ? " " : line,
-      path.call(print, "right")
-    ]);
+    const shouldInline = shouldInlineLogicalExpression(node);
+    const lineBeforeOperator = node.operator === "|>";
+
+    const right = shouldInline
```

Comment:
This change prints the `|>` operator on the LHS. All other operators are printed before the line break.